source/opt/struct_cfg_analysis.h

```diff
@@ -103,6 +119,7 @@ class StructuredCFGAnalysis {
     uint32_t containing_construct;
     uint32_t containing_loop;
     uint32_t containing_switch;
+    bool in_continue;
```

Comment:
The comment hasn't been updated for `|containing_switch|` and `|in_continue|`

Reply:
Done